ui/redux/actions/settings.js

```diff
@@ -42,6 +81,19 @@ export function doSetDaemonSetting(key, value) {
   };
 }
 
+export function doCacheCustomWalletServers(servers) {
+  return {
+    type: ACTIONS.WALLET_SERVERS_CACHED,
+    data: servers,
+  };
+}
+
+export function doGetDaemonStatus() {
+  return dispatch => {
+    return Lbry.status().then(settings => settings);
```

Comment:
This doesn't need to be in an action creator if its not dispatching an action. Just import `Lbry` wherever it's being used.

Reply:
Chose to make it store status next to daemonSettings instead. We can call and poll it the same way we use daemonSettings in the future like we talked about.